code/game/objects/items/weapons/bee_briefcase.dm

```diff
@@ -26,9 +26,9 @@
 			. += "<span class='warning'>There are [bees_left] bees still inside in briefcase!</span>"
 		else
 			. += "<span class='danger'>The bees are gone... Colony collapse disorder?</span>"
+	if(isAntag(user))
+		. += "<span class='warning'>A briefcase filled with deadly bees, you should inject this with a syringe of your own blood before opening it. Exotic blood cannot be used.</span>"
```

Comment:
I suggest to change this to `notice` class and tell the user _why_ they need to inject it with blood (and what it does if they fail to do so).